src/imports.rs

```diff
@@ -168,6 +241,17 @@ impl UseTree {
         Some(result)
     }
 
+    // FIXME: Use correct span?
```

Comment:
Why is the span incorrect?

Reply:
The span could be wrong when we are using the span from list imports. It should not matter, though, since we have already extracted comments around use items at this point.

Reply:
Could you add this detail to the comment please?